Formula/postgresql@10.rb

```diff
@@ -4,6 +4,7 @@ class PostgresqlAT10 < Formula
   url "https://ftp.postgresql.org/pub/source/v10.21/postgresql-10.21.tar.bz2"
   sha256 "d32198856d52a9a6f5d50642ef86687ac058bd6efca5c9ed57be7808496f45d1"
   license "PostgreSQL"
+  revision 2
```

Comment:
```suggestion
  revision 1
```

Reply:
Thanks - will commit this after CI finishes